contract/contracts/beacons/FactoryBeacon.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./UserBeaconProxy.sol";
+
+contract FactoryBeacon is UpgradeableBeacon {
+  event BeaconProxyCreated(address indexed proxy, address indexed deployer);
+
+  constructor(address implementation) UpgradeableBeacon(implementation, msg.sender) {}
+
```

Comment:
I have a problem from this line, but i think it's coming from the model @farrellh1 create. 

The implementation name should be implementationAddress so we no what type of value we are using.